b2sdk/api_config.py

```diff
@@ -34,7 +33,7 @@ def __init__(
         self.http_session_factory = http_session_factory
         self.install_clock_skew_hook = install_clock_skew_hook
         self.user_agent_append = user_agent_append
-        self.raw_api_class = _raw_api_class
+        self.raw_api_class = _raw_api_class or B2RawHTTPApi
```

Comment:
I think it should be `self.DEFALT_RAW_API_CLASS` for future proofing like we did earlier in this PR